plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryAvroConnectorTest.java

```diff
@@ -51,7 +51,6 @@ protected QueryRunner createQueryRunner()
     {
         return BigQueryQueryRunner.builder()
                 .setConnectorProperties(ImmutableMap.<String, String>builder()
-                        .put("bigquery.arrow-serialization.enabled", "false")
```

Comment:
> Enable arrow serialization in TestBigQueryAvroConnectorTest

I don't understand the motivation. This test should use Avro serialization. Please revert.

Reply:
removed